mars-registration-form/src/Components/Form.js

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import listOfCountries from "./listOfCountries";
+
+class Form extends React.Component{
+
+state = {
+    firstName: "", 
+    lastName: "", 
+    dateOfBirth: "", 
+    country: "", 
+    dietaryOptions: "",
+    question: "",
+    formCompleted: false
+}
+handleChange = (event) => {
+    this.setState({
+        [event.target.name]: event.target.value
+    })
+} 
+
+formSubmitted = () => {
+    for(let key in this.state){
+        if(this.state[key] !== ""){
+            return (
+                <div>FORM COMPLETED</div>
+            )
+        }
+    }
+}
```

Comment:
Wouldn't this just return if any of the keys were complete? Once you return from a function its over.